lib/utils.js

```diff
@@ -137,3 +137,20 @@ exports.noOp = function (_cb) {
 	}
 
 };
+
+/**
+ * Return true if object is empty else false
+ * @param {Object} obj   Object to be tested if it is empty or not
+ * @return {Bollean}
+ */
+exports.isEmptyObject = function (obj) {
```

Comment:
Not sure why you need this function, we normally use `Object.keys(obj).length` to determine is an object has any keys.